src/app/hooks/useViewTracker/index.tsx

```diff
@@ -101,7 +101,7 @@ const getComponentViewTracker = (eventTrackingData?: EventTrackingData) => {
 
             optimizely.track(
               optimizelyMetricNameOverride
-                ? `${optimizelyMetricNameOverride}_views`
+                ? `${componentName}-views`
                 : 'component_views',
```

Comment:
Now that we're removing `optimizelyMetricNameOverride` completely, let's update this: 

```suggestion
            optimizely.track(`${componentName}-views`,
```